thirdeye/thirdeye-frontend/app/pods/rootcause/controller.js

```diff
@@ -1,56 +1,70 @@
 import Ember from 'ember';
-import { makeIterable, filterObject, filterPrefix, toBaselineUrn, toCurrentUrn } from 'thirdeye-frontend/helpers/utils';
+import { filterObject, filterPrefix, toBaselineUrn, toCurrentUrn } from 'thirdeye-frontend/helpers/utils';
 import EVENT_TABLE_COLUMNS from 'thirdeye-frontend/mocks/eventTableColumns';
 import config from 'thirdeye-frontend/mocks/filterBarConfig';
+import SHA256 from 'cryptojs/sha256';
```

Comment:
since this is an ember addon, can we just do `import CryptoJS from 'cryptojs';` as the doc describes here: 
https://www.npmjs.com/package/ember-cryptojs-shim

Reply:
fixed. both ways are mentioned in the docs